docker/Dockerfile.pdo-build

```diff
@@ -101,6 +102,13 @@ ENV PDO_DEBUG_BUILD=${PDO_DEBUG_BUILD}
 ARG PDO_INTERPRETER=gipsy
 ENV PDO_INTERPRETER=${PDO_INTERPRETER}
 
+# - web-assembly/wasm/wawaka
+#   - Configure WAMR source
+ARG WAMR=WAMR-03-05-2020
+RUN cd /project/pdo/src/private-data-objects/interpreters/wasm-micro-runtime \
+&& git checkout ${WAMR}
```

Comment:
a trailing \ is missing (and for stylistic reasons might be nice to preceed && with some spaces?)